gamification-crowdin-services/src/main/java/io/meeds/gamification/crowdin/dao/WebHookDAO.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * This file is part of the Meeds project (https://meeds.io/).
+ * Copyright (C) 2020 - 2023 Meeds Association contact@meeds.io
```

Comment:
```suggestion
 * Copyright (C) 2023 Meeds Lab contact@meedslab.com
```

Reply:
Thanks. Just to confirm, the year is 2023 not 2024?

Reply:
Yes, 2023